frontend/icons/download-dark.svg

```diff
@@ -0,0 +1,4 @@
+<symbol id="download-dark" viewBox="0 0 25 25">
```

Comment:
Have the icons been piped through SVGOMG?

Reply:
Yep :)